### ISSUE TITLE
Check if error is a string or an object

### ISSUE THREAD

**Issue by dvh** [renedx/node-moneybird#1]
Posting a `contact` with a duplicate `customer_id` results in the following API response:

**422: Unprocessable entity**
```
{
  "error": {
    "customer_id": [
      "already exists"
    ]
  }
}
```

Unlike other error messages from the Moneybird API, the `error` is an object instead of a string here. This breaks your code, because it seems that https://github.com/renedx/node-moneybird/blob/master/src/libs/https.js#L31 assumes `error` is always a string.

**Comment by rdzar** [comment]
Thanks for your detailed bug-report! This has been fixed in `1.0.2` on NPM.

An example of usage can be found here:
https://github.com/renedx/node-moneybird#example-create-contact-with-fields-error

**Comment by dvh** [comment]
Great, thanks!
